data/random-teams.js

```diff
@@ -1835,9 +1835,16 @@ class RandomTeams extends Dex.ModdedDex {
 				pokemon.push(set);
 
 				if (pokemon.length === 6) {
-					// Set Zoroark's level to be the same as the last Pokemon
 					let illusion = teamDetails['illusion'];
```

Comment:
Why isn't all this just after the `break`?

Reply:
I don't know, man. I'm not in this part of the codebase to ask questions, I'm here to fix crashes and help out the poor souls stuck with a 5-mon monoice team

Reply:
I mean, fair, but since you're rewriting this block anyway, you might as well move it to be after the iteration. The less heavily-indented code we have, the better.